Web/Edubase.Web.UI/Assets/Scripts/GiasVueComponents/GiasRadio.vue

```diff
@@ -15,19 +15,16 @@
 <script>
     export default {
       name: "giasRadio",
-      model: {
-        prop: 'checked',
-        event: 'change'
-      },
       props: {
         value: String,
         labeltext: String,
         binding: String,
         name: String,
+        checkedValue: String,
       },
       methods: {
         emitValue: function(value) {
-          this.$emit('input', value);
+          this.$emit('update:checkedValue', value);
         }
       },
       computed: {
```

Comment:
I've not specifically tested/checked this.

A pointer to the pages this loads on would be appreciated :)

Reply:
I believe this was for the radio buttons not working as expected: 
<img width="583" alt="image" src="https://github.com/user-attachments/assets/e18017df-5fe0-4e18-9d70-97b68350ce79">